src/pages/Homepage/Categories/Categories.jsx

```diff
@@ -1,5 +1,24 @@
+import "./Categories.css";
+import Pasta from "../../../assets/pasta.jpg";
+import Burger from "../../../assets/burger.jpg";
+import Pizza from "../../../assets/pizza.png";
+import Pancake from "../../../assets/pancake.jpg";
+import Salad from "../../../assets/salad.jpg";
+import { Card } from "../../../components/Card/Card";
```

Comment:
I think that naming should be done like PastaImg or PastaImage so that its understandable that its an image. Similarly do for other images as well.

Reply:
Good Point! Will be implementing this